app/util/appUtil.js

```diff
@@ -443,6 +459,11 @@ const getMetadataResponse = async (tokenId, metadataKey, res) => {
     return
   }
 
+  if (metadataValue.tokenId) {
+    res.set('content-type', 'text/plain')
+    res.status(200).send(metadataValue.tokenId.toString())
+  }
```

Comment:
do you need to return  inside this block?

Reply:
yes, added